src/bluesearch/widgets/search_widget.py

```diff
@@ -896,18 +895,18 @@ def _create_saving_checkboxes(self, article_id, paragraph_id):
 
         return chk_article, chk_paragraph
 
-    def _cb_bt_pdf_report_article_saver(self, change_dict):
+    def _cb_bt_make_report_article_saver(self, change_dict):
         """Create the saved articles report."""
         with self.widgets["status"]:
             print()
-            print("Creating the saved results PDF report... ")
-            self.article_saver.pdf_report()
+            print("Creating the saved results report... ")
+            self.article_saver.make_report()
```

Comment:
`make_report()` returns the file path of the saved report - we could actally print it for the user as well, something like
```python
report_path = self.article_saver.make_report()
print(f"The report was saved to {report_path.resolve()}")
```

Reply:
Good idea! Done in 2dd1414.